src/librustc_typeck/check/mod.rs

```diff
@@ -155,6 +155,17 @@ use self::method::{MethodCallee, SelfSource};
 pub use self::Expectation::*;
 use self::TupleArgumentsFlag::*;
 
+#[macro_export]
```

Comment:
I think we can drop the `macro_export` here if we place this above the `mod` declarations (might already be the case), would be nice to avoid letting other crates use it perhaps? Not sure.

Reply:
It didn't work :(